addons/aircraft/XEH_PREP.hpp

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Yes, because people delete them with the same argument as Neviothr made. Do what you like best. It really doesn't matter.